### ISSUE TITLE
Rename createMember function to addMember and Update Permission to Allow Admins

### ISSUE THREAD

**Issue by pranshugupta54** [PalisadoesFoundation/talawa-api#1993]
**Description:**

**Issue:**
The `createMember` function in the mutation resolvers is named in a way that might cause confusion about its purpose. Additionally, it currently restricts the ability to add members to only superAdmin users, limiting flexibility in managing organization memberships.

**Proposed Solution:**
1. Rename the `createMember` function to `addMember` to better reflect its purpose of adding a member to an organization.
2. Update the permission logic to allow both `superAdmin` users and organization `admins` to add members. This change can provide more flexibility in managing organization memberships.

**Expected Outcome:**
- The function will be renamed to `addMember`, making its purpose clearer.
- Both superAdmin users and organization admins will be able to add members to an organization, enhancing flexibility in managing memberships.

**Currently:**
<img width="696" alt="image" src="https://github.com/PalisadoesFoundation/talawa-api/assets/76090263/f4c5b27f-922c-48d8-b59a-614959df20f3">

**Reference:**
- Needed in PalisadoesFoundation/talawa-admin#1719

**Comment by Anubhav-2003** [comment]
@palisadoes I would like to work on this issue.

**Comment by palisadoes** [comment]
I accidentally merged before testing was completed.

Please submit another PR against this issue to verify all tests pass

**Comment by Anubhav-2003** [comment]
Ok sir

**Comment by Anubhav-2003** [comment]
@palisadoes Respected sir, I have raised another PR as per your request, but after my PR got merged, I pulled the latest upstream, and ran tests. All the tests are passing in the updated develop branch. Also the merged PR has not been reverted, so my latest re-PR is showing no files as changed. But it is passing all the tests signifying the current develop passed all of them. So we can keep the current merge sir.

Thank You.
